<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameStatusClass</class>
 <widget class="QDialog" name="GameStatusClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>673</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>882</width>
    <height>673</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>925</width>
    <height>673</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>网络五子棋 - 游戏中....</string>
  </property>
  <property name="windowIcon">
   <iconset resource="client.qrc">
    <normaloff>:/Client/Resources/exe.ico</normaloff>:/Client/Resources/exe.ico</iconset>
  </property>
  <widget class="QLabel" name="gameStatus_client_LB">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>640</y>
     <width>631</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>640</y>
     <width>640</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>241</width>
     <height>661</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>自己</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>昵称:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="addr_me_client_LE">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(243, 243, 243);</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>颜色:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="color_me_client_LE">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(243, 243, 243);</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>战绩:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="record_me_client_LE">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(243, 243, 243);</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>对手</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>昵称:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="addr_rivar_client_LE">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(243, 243, 243);</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>颜色:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="color_rivar_client_LE">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(243, 243, 243);</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>战绩:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="record_rivar_client_LE">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(243, 243, 243);</string>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>聊天</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTextEdit" name="chatLog_client_TE">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(243, 243, 243);</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLineEdit" name="sendwords_client_LE">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="send_client_BTN">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>发送</string>
           </property>
           <property name="default">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="undo_client_BTN">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>悔棋</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="lose_client_BTN">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>认输</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="gameReady_client_BTN">
        <property name="text">
         <string>准备</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quit_client_BTN">
        <property name="text">
         <string>退出</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>send_client_BTN</tabstop>
  <tabstop>sendwords_client_LE</tabstop>
  <tabstop>gameReady_client_BTN</tabstop>
  <tabstop>undo_client_BTN</tabstop>
  <tabstop>lose_client_BTN</tabstop>
  <tabstop>quit_client_BTN</tabstop>
  <tabstop>addr_me_client_LE</tabstop>
  <tabstop>record_me_client_LE</tabstop>
  <tabstop>addr_rivar_client_LE</tabstop>
  <tabstop>record_rivar_client_LE</tabstop>
  <tabstop>chatLog_client_TE</tabstop>
 </tabstops>
 <resources>
  <include location="client.qrc"/>
 </resources>
 <connections/>
</ui>
